### PR TITLE
ESR Notifications and Short Description fields

### PR DESCRIPTION
**Pull request by sarwarahmed1**
Reviewed

### CODE REVIEW

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRNotificationStepsImplementation.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package ServiceNow.ESR.Playwright.StepsImplementation;
+
+import ServiceNow.ESR.Playwright.Pages.CreateESRPage;
+import com.microsoft.playwright.FrameLocator;
+import com.microsoft.playwright.options.AriaRole;
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRNotificationStepsImplementation.java:6]: please remove the empty line 6

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRShortDescriptionStepsImplementation.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package ServiceNow.ESR.Playwright.StepsImplementation;
+
+import ServiceNow.ESR.Playwright.Pages.CreateESRPage;
+
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRShortDescriptionStepsImplementation.java:4]: please remove empty line 4

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
please review the comments. thanks!

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRNotificationStepsImplementation.java (hunk 1 above)
Comment (sarwarahmed1) [reply src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRNotificationStepsImplementation.java]: Resolved

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRShortDescriptionStepsImplementation.java (hunk 2 above)
Comment (sarwarahmed1) [reply src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRShortDescriptionStepsImplementation.java]: Resolved

**Review by katm007 — APPROVED** [review]
Reviewed and approved!

**Review by lfcabrera14 — APPROVED** [review]
Approved

**Review by alenaminsk — APPROVED** [review]

**Pull request merged**